crowdsec-docs/unversioned/getting_started/introduction.md

```diff
@@ -0,0 +1,13 @@
+---
+id: intro
+title: Introduction
+sidebar_position: 1
+---
+
+Welcome to the getting started guide for CrowdSec.
+
+This guide will walk you through the installation of your first CrowdSec Security Engine and the deployment of your first [Remediation Component](/bouncers/intro.md).
+
```

Comment:
We should use this page to remind the user on the objective of the software and what it achieves.

Reply:
Up to you I believe the sentence itself describes the outcome